README.rst

```diff
@@ -1,5 +1,16 @@
-drydock: a flexible manifest builder for Open edX
-=================================================
+Drydock
+=======
+
+Drydock is a tool that aims to provide a set of plugins that edunext considers useful to manage openedx installations.
+
+
+- A set of Kubernetes Jobs that replace the current tutor jobs with `ArgoCD Sync Waves <https://argo-cd.readthedocs.io/en/stable/user-guide/sync-waves/>`_ that allow for a more controlled deployment of openedx.
+- A set of Kustomization overrides adding `ArgoCD Sync Waves <https://argo-cd.readthedocs.io/en/stable/user-guide/sync-waves/>`_ annotations to needed additional resources such as debug, workers or hpa.
+- Backup cronjobs that allow backup of the MySQL and MongoDB databases.
+- Integration of New Relic monitoring
+- Add flowers deployment for Celery
```

Comment:
Could you please change all the references to `flowers` to `flower`? It's not like we have multiple instances of the [Flower](https://flower.readthedocs.io/en/latest/) service

Reply:
I think we would need to mention in the release notes that DRYDOCK_FLOWERS was moved to DRYDOCK_FLOWER